ci/azure-pipelines.yml

```diff
@@ -100,13 +101,15 @@ stages:
               kernel_ver=$(dpkg -l | grep 'linux-headers-.*-generic' | awk '{print $2}')
               ./configure --enable-gtest --with-kerneldir=/usr/src/${kernel_ver}
               make
-            displayName: Build Ubuntu
+            displayName: Build on Ubuntu
             condition: contains(variables['build_container'], 'ubuntu')
 
-  - stage: Test
+  - stage: VMs
+    displayName: Build & Test on VMs
     dependsOn: Codestyle
     jobs:
       - job: Test
+        timeoutInMinutes: 240
```

Comment:
would 30 or  60 minutes would be already large enough?

Reply:
Compiling GCC alone takes about an hour.
More time for existing and future tests + some buffer.

Reply:
makes sense